src/language-server/project/client.ts

```diff
@@ -28,7 +28,7 @@ import {
   DiagnosticSeverity,
 } from "vscode-languageserver/node";
 import LZString from "lz-string";
-import { stringifyUrl } from "query-string";
```

Comment:
New versions of `"query-string"` are ESM only and I really don't want to touch the build process of the Extension right now.